src/main/webapp/script.js

```diff
@@ -260,3 +295,137 @@ function checkPercentage(headerColumn) {
   }
   return false;
 }
+
+// Returns an object containing all of the relevant data in order
+// to render the geoJson map of the counties.
+function getMapsData(censusDataArray) {
+  const countyToPopMap = new Map();
+  const populationsList = [];
+  Array.prototype.forEach.call(censusDataArray, (county) => {
+    if (county[0] !== 'NAME') {
+      // The current county strings are in a layout like this:
```

Comment:
this is very brittle. Is this the only way the census data identifies the county?

Reply:
How does the amcharts display code handle this situation?

Reply:
Fixed to .slice(1) in order to get rid of the header when traversing the census data.